### ISSUE TITLE
Schlage connect stopped working

### ISSUE THREAD

**Issue by FabLabEP** [nfarina/homebridge#705]
Schlage connect touchscreen lock was working fine, but decided to stop responding.  Can control via Wink app and via Siri if part of a scene.  However, all direct calls to lock/unlock return "Sorry, I didn't hear back."

Tried deleting persist/accessories, resetting HomeKit configuration, rebooting Wink hub, etc...etc... but the lock just won't respond.

Any ideas?

**Comment by jclarke0000** [comment]
Also having issues with Schlage Connect.  Locks can be controlled using Elgato Eve, but issuing a command via Siri does not work.  Other items controlled via Wink continue to work with Siri (lights, garage door).

Not sure if it makes a difference, but I have two Schlage Connect locks.  Both were previously working via Siri.  The last time I remember successfully using Siri to control the locks was on July 1st.

**Comment by labdaemons** [comment]
As a workaround I created scenes called "Unlock Front Door" and "Unlock Front Door".  They work via Siri instead of a direct call.

**Comment by jclarke0000** [comment]
hmmm... this seems oddly familiar to an issue I ran into recently with Lutron Caestta, where I couldn't control individual lights.  Something had changed in HomeKit such that I could no longer target individual lights by saying "Turn on the Front Entrance lights" if I included the word "lights".  If I instead said "Turn on the Front Entrance" it worked.  Oddly enough, targeting groups of lights (example: "Turn on the Dining Room lights") continued to work as before.  I needed to add the word "Lights" to the Siri name in order to target individual lights.

In this case, I haven't figured out what the offending word is for the locks, but I'm guessing it is something similar.  Likewise with the lights, if I target them as a group (e.g.: "Lock all the doors" or "Are the doors locked?") everything works normally.

**Comment by jclarke0000** [comment]
OK more weirdness, but I got this to work.  I needed to use the ConnectSense app to do so.  In the device settings, there were two Service / Siri Names listed for the locks, both of them were the same (e.g.: "Back Door" listed twice).  I renamed the second one to something different entirely, and ALSO renamed the device in ConnectSense to something different (e.g.: Device name "Front Door Lock", Service / Siri Names: "Front Door", "Whatever").  Now I ca control my locks once again.
